tools/orca/src/stats.rs

```diff
@@ -141,15 +141,15 @@ impl DataCollector for BasicStatistics {
             return Err(Error::InvalidState);
         }
 
-        if writeop_times.is_empty() {
-            error!("For some weird reason no write operations were recorded, exiting...");
-            return Err(Error::InvalidState);
-        }
+        // if writeop_times.is_empty() {
```

Comment:
Why is this disabled?

Reply:
Because when running a write only / read only test we expect to have 0 read events / write events respectively. Idk if you like my workaround to avoid panicking tho, I'm not sure I do either 🥲